lib/osf-components/addon/components/registries/schema-block-renderer/component.ts

```diff
@@ -14,14 +13,14 @@ import template from './template';
 export default class SchemaBlockRenderer extends Component {
     // Required params
     schemaBlock!: SchemaBlock;
-    changeset!: Changeset;
+    blockStrategy!: Component;
 
     // Optional params
     disabled: boolean = defaultTo(this.disabled, false);
     shouldShowMessages: boolean = defaultTo(this.shouldShowMessages, true);
```

Comment:
changeset under optional params

Reply:
this also looks like a leftover comment from an earlier version. changesets and pageResponses are passed into the mappers now